unittests_launcher.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import pytest
+import sys
+
+sys._pwndbg_unittest_run = True
+
+# If you want to debug tests locally, add '--pdb' here
+args = ['./tests/', '--pdb', '-vvv', '-s', '--showlocals', '--color=yes']
```

Comment:
We should not invoke with `--pdb` unless it's provided on the command-line,  I'd recommend extending with `sys.argv` or something.  The relative path might also bite us at some point.

Reply:
Is this doable to pass `--pdb` to `gdb ...`?